flipperdevices/flipperzero-firmware
# [AVR_ISP]: add AVR ISP Programmer FAP

# What's new

- AVR_ISP: add AVR ISP Programmer FAP

# Verification 

- connect AVR chip according to 
      MISO - PA6
      MOSI - PA7
      SCK - PB3
      RESET- PB2
      CLOCK - RA4 (as needed)
- Run FAP AVR ISP Programmer, if everything is connected correctly, the chip should be determined, and a new com port should appear in the system
- Run a program such as "ardude" and check the read / write AVR chip
   exemple : 
   avrdude.exe -p m328p -c stk500v1 -P COM10  -U lock:r:lock.hex:r -U hfuse:r:hfuse.hex:r -U lfuse:r:lfuse.hex:r -U efuse:r:efuse.hex:r
![image](https://user-images.githubusercontent.com/85568270/225079952-d0980717-7062-4618-b684-05c15ea15852.png)




# Checklist (For Reviewer)
